float64[] coord
float32 speed
float64 heading
uint64 timestamp
uint64 source_time

